lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -3948,12 +3957,21 @@ void SwiftASTContext::ValidateSectionModules(
 
   Status error;
 
+  Progress progress(
+      llvm::formatv("Loading swift module {0}",
+                    module.GetFileSpec().GetFilename().AsCString()),
+      module_names.size());
+
+  size_t completion = 0;
+
   for (const std::string &module_name : module_names) {
     SourceModule module_info;
     module_info.path.push_back(ConstString(module_name));
     if (!GetModule(module_info, error))
       module.ReportWarning("unable to load swift module \"%s\" (%s)",
                            module_name.c_str(), error.AsCString());
+    else
```

Comment:
By putting this in the else branch, we might report less events than `module_names.size()`. Why not increment the value even if the module couldn't be loaded?

Reply:
Good call!